lib/block_view_validator_test.go

```diff
@@ -629,3 +631,260 @@ func _testValidatorRegistrationWithDerivedKey(t *testing.T) {
 	require.NoError(t, mempool.universalUtxoView.FlushToDb(blockHeight))
 	_executeAllTestRollbackAndFlush(testMeta)
 }
+
+func _testGetTopValidatorsByStake(t *testing.T, flushToDB bool) {
+	var validatorEntries []*ValidatorEntry
+	var err error
+
+	// Initialize balance model fork heights.
+	setBalanceModelBlockHeights()
+	defer resetBalanceModelBlockHeights()
+
+	// Initialize test chain and miner.
+	chain, params, db := NewLowDifficultyBlockchain(t)
+	mempool, miner := NewTestMiner(t, chain, params, true)
+
+	// Initialize PoS fork height.
+	params.ForkHeights.ProofOfStakeNewTxnTypesBlockHeight = uint32(1)
+	GlobalDeSoParams.EncoderMigrationHeights = GetEncoderMigrationHeights(&params.ForkHeights)
+	GlobalDeSoParams.EncoderMigrationHeightsList = GetEncoderMigrationHeightsList(&params.ForkHeights)
+	chain.snapshot = nil
```

Comment:
can you remind me why we set the snapshot to nil here?

Reply:
Can't remember where, but this was a potential lead for why some tests were intermittently failing. Piotr recommended adding this to prevent snapshotting from interfering with tests. Just an extra safety check. I removed and tests still pass. Gonna leave off and we'll see if it ever causes us issues.